src/core/util/util.js

```diff
@@ -46,6 +46,19 @@ angular.module('material.core')
             return results;
           },
 
+          /**
+           * Calculate the positive scroll offset
+           */
+          scrollTop : function(element) {
+            element = angular.element(element || $document[0].body);
+
+            var body = (element[0] == $document[0].body) ? $document[0].body : undefined;
+            var scrollTop = body ? body.scrollTop + body.parentElement.scrollTop : 0;
+
+             // Calculate the positive scroll offset
+            return scrollTop || Math.abs(element[0].getBoundingClientRect().top);
```

Comment:
You should check if this works as expected when the page has been pinch-zoomed in either IE or Chrome. 
There is an [issue](https://code.google.com/p/chromium/issues/detail?id=496285) where some positional properties of the body are computed in a different context from the output of `getBoundingClientRect`.